### DISCUSSION TITLE
Question DLC

### DISCUSSION THREAD

**Discussion by ElkacemDZ** [L-Sherry/French-CC#9] [Q&A]
FR : 
Bonjour, aujourd'hui je vous écris avant tout pour vous remercier. J'ai rejoint le serveur Discord de la communauté en espérant vous remercier directement mais hélas l'occasion ne ce n'est pas présentée… donc merci ! Merci énormément, pour être le plus honnête avec vous, j'ai un handicap qui m'empêche l'apprentissage de langue étrangère. Les personnes comme vous alors me permettent d'une certaine manière de retrouver des libertés. 
En l'occasion de ces remerciements, j'ai hésité aussi à vous poser ma question. .je voudrais pas que cela gâche mes remerciements mais je tente ! Avez-vous prévu dans le futur de traduire également le DLC ? Il est très récent et je vous demande pas alors si demain une traduction complète et parfaite sera faites, seulement si dans l'avenir vous avez le projet ou bien que vous avez commencé ....
Merci encore
Cordialement, un joueur heureux

EN :
Hello, today I am writing to you first of all to thank you. I joined the community's Discord server hoping to thank you directly but unfortunately the opportunity is not presented ... so thank you! Thank you very much, to be honest with you, I have a disability that prevents me from learning a foreign language. People like you then allow me in a certain way to find freedoms.
On the occasion of these thanks, I also hesitated to ask you my question. .II wouldn't want this to make a stain on my thanks but I try! Do you have plans in the future to translate the DLC as well? It is very recent and I am not asking you then if tomorrow a complete and perfect translation will be done, only if in the future you have the project or have started ...
thanks again
Sincerely, a happy player
PS : translation done with a translator, sorry for the calamitous English

**Comment by L-Sherry** [comment]
Vu la taille du DLC, à mon avis, ça devrait pas être grand chose à traduire.

Par contre, avant, il faut que j'y joue, et que je le finisse. Et que j'explore un peu toutes les possibilités avant de commencer à traduire, parce que traduire est un bon moyen de tout dévoiler et de se gâcher le jeu...

**Reply by ElkacemDZ** [reply]
Je vois ! Prenez votre temps pour vous amusez !
Depuis votre message j'ai pris le DLC, j'ai remarqué que certain message précédemment traduit ne le sont plus, surement faute des dialogue qui se sont rajouté dans le code du jeu...
Je vous remercie et vous souhaite bon jeu!
